go.mod

```diff
@@ -23,14 +23,15 @@ require (
 	github.com/iceber/iouring-go v0.0.0-20220609112130-b1dc8dd9fbfd
 	github.com/jedib0t/go-pretty/v6 v6.3.1
 	github.com/json-iterator/go v1.1.12
+	github.com/labstack/gommon v0.4.0
 	github.com/linkall-labs/embed-etcd v0.1.1
 	github.com/linkall-labs/vanus/client v0.5.1
 	github.com/linkall-labs/vanus/observability v0.5.1
 	github.com/linkall-labs/vanus/pkg v0.5.1
 	github.com/linkall-labs/vanus/proto v0.5.1
 	github.com/linkall-labs/vanus/raft v0.5.1
 	github.com/ncw/directio v1.0.5
-	github.com/ohler55/ojg v1.14.5
+	github.com/oliveagle/jsonpath v0.0.0-20180606110733-2e52cf6e6852
```

Comment:
why change back to `github.com/oliveagle/jsonpath`?

Reply:
The reason changed to `github.com/ohler55/ojg v1.14.5` isn't `github.com/oliveagle/jsonpath` which doesn't support array, rather than `github.com/oliveagle/jsonpath` has many errors, you can see the comparison in https://cburgmer.github.io/json-path-comparison/